packs/actions/long-jump.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "interaction",
         "description": {
-            "value": "<p>You Stride, then attempt a @Check[type:athletics|dc:15] check to make a long jump in the direction you were Striding. If you didn't Stride at least 10 feet, you automatically fail your check. The GM might increase or decrease this DC depending on the situation.</p>\n<hr />\n<p><strong>Success</strong> You @UUID[Compendium.pf2e.actionspf2e.Item.Leap] a distance equal to your check result rounded down to the nearest 5 feet. You can't jump farther than your land Speed.</p>\n<p><strong>Failure</strong> You make a normal horizontal Leap.</p>\n<p><strong>Critical Failure</strong> You make a normal horizontal Leap, then fall and land @UUID[Compendium.pf2e.conditionitems.Item.Prone].</p>"
+            "value": "<p>You Stride, then attempt a DC 15 <span data-pf2-action=\"longJump\" data-pf2-glyph=\"D\">Athletics</span> check to make a long jump in the direction you were Striding. If you didn't Stride at least 10 feet, you automatically fail your check. The GM might increase or decrease this DC depending on the situation.</p>\n<hr />\n<p><strong>Success</strong> You @UUID[Compendium.pf2e.actionspf2e.Item.Leap] a distance equal to your check result rounded down to the nearest 5 feet. You can't jump farther than your land Speed.</p>\n<p><strong>Failure</strong> You make a normal horizontal Leap.</p>\n<p><strong>Critical Failure</strong> You make a normal horizontal Leap, then fall and land @UUID[Compendium.pf2e.conditionitems.Item.Prone].</p>"
```

Comment:
I believe you can inject the DC with `data-pf2-dc=\"15\"`

Reply:
You can